### DISCUSSION TITLE
Beginner Here, Profile Mode doubt

### DISCUSSION THREAD

**Discussion by Astrodevil** [adityatelange/hugo-PaperMod#707] [Q&A]
As mentioned in wiki of this project that to use Profile Mode paste this code in config file... but when I am doing so it gives build error. please clear my doubt, do I have to paste the given script in config.toml file or by creating config.yml file

Is there any video guide to use this theme properly? that would be helpful

**Comment by apurvakhatri** [comment]
It must be an image location error. You need to set "imageUrl" key to some value. Probably the name of your image which is kept in themes/PaperMod/images.

**Comment by iamalexblue** [comment]
Same issue, how did you fix that bro?
